### ISSUE TITLE
Local mqtt data

### ISSUE THREAD

**Issue by oeyhaga** [ottopaulsen/MMM-Tibber#44]
Hey. 
I use this module every day. But im thinking of moving my tibber pulse over to local mqtt and home assistant with amshan.  Would it be possible to make this module collect data from home assistant instead of tibber cload?

**Comment by ottopaulsen** [comment]
You are welcome to try :-)

**Comment by scientific-method-course** [comment]
The disadvantage of MQTT data is they do not carry the history with them (only provide intantaneous data). I went a slightly different route (which could be combined with reading data from MQTT) in two steps:
- collect the history in a local CSV file on the machine where your run the MagicMirror (in my case I get them from through the API of my Domoticz home automation server, but  this should be possible from Homeassistent as well, I guess). The Python script that collects those data and puts them in a useful format is running every 10 minutes.
- let the MMM-Tibber read  the consumption data from that CSV file and use it to complement the missing consumption data (= consumption data from the current day). At the moment, I only needed to modify node_helper.js for this. But if I want to expose some of the settings (location and format of the CSV file) to be included in the configuration file, I would need to modify somewhat more.

The code is not yet mature enough to share, I think (I'm an experienced programmer, but Javascript is one of those languages that I'm just about to add to my list of skills).
